core/src/test/java/ai/timefold/solver/core/impl/domain/common/accessor/gizmo/GizmoMemberAccessorFactoryTest.java

```diff
@@ -36,8 +36,8 @@ void setup() {
     void testReturnedMemberAccessor() throws NoSuchMethodException {
         Method member = TestdataEntity.class.getMethod("getValue");
         MemberAccessor memberAccessor =
-                GizmoMemberAccessorFactory.buildGizmoMemberAccessor(member, PlanningVariable.class, true,
-                        new GizmoClassLoader());
+                GizmoMemberAccessorFactory.buildGizmoMemberAccessor(member, PlanningVariable.class,
+                        AccessorInfo.of(true, false), new GizmoClassLoader());
```

Comment:
`AccessorInfo.of(true, false)` says nothing.
`AccessorInfo.withReturnValueAndNoArguments()` says everything.

Reply:
Arguably, `AccessorInfo` can be an enum.